.github/actions/verify-gha-refs/Check-Called-GHA-Refs.ps1

```diff
@@ -0,0 +1,44 @@
+param(
+    [String[]] $CalledRepoBaseIncludeList, 
+    [String[]] $PathIncludeList,
+    [String[]] $FileIncludeList,
+    [String] $ExpectedRef,
+    [String] $GitHubRepository,
+    [String] $GitHubRefName
+)
+
+if ($CalledRepoBaseIncludeList.Count -eq 0) {
+    exit 0 # Nothing to check because array is empty.
+}
+
+$CalledRepoBaseIncludeList = $CalledRepoBaseIncludeList.ForEach({ "uses:\s*" + $_ })
+
+$mismatchRefs = Get-ChildItem -Path $PathIncludeList -Include $FileIncludeList -Force -Recurse | 
+Select-String -Pattern $CalledRepoBaseIncludeList | 
+Select-String -Pattern $ExpectedRef -NotMatch
+
+if ($mismatchRefs.Count -gt 0)
+{
+        "These called worflows and actions do not match expected ref '$ExpectedRef'." >> $env:GITHUB_STEP_SUMMARY
```

Comment:
```suggestion
        "These called workflows and actions do not match expected ref '$ExpectedRef'." >> $env:GITHUB_STEP_SUMMARY
```

Reply:
Szabolcs, can you please check the spell-checking configuration? This should've been caught by the action.

Reply:
It ran, and I think it saw this file. So good question what happened.

Reply:
I fixed the spelling error. I think files with a .ps1 extension are excluded from spelling check based on [this excludes file](https://github.com/Lombiq/GitHub-Actions/blob/dev/.github/actions/spelling/excludes.txt#L22) in the spelling action. 

Do note that I removed that line from the excludes.txt file in this branch to demonstrate how changing any file under and action can trigger the verify-gha-refs workflow to find it.

Reply:
Really good catch, thank you!